src/components/UseCurrentWord.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import getRandomWord from './GetRandomWord';
+import {saveGameState} from './LoadSaveGame';
+
+
+/**
+ * React does not know that it needs to re-render the answer component
+ * unless I use the react state. So the displayed answer will only change when 
+ * React watches for the state of the answer variable to change.
+ */
+
+/**
+ * Used to get the correct words from the category in the bank.
+ */
+const CATEGORIES = new Map([
+    ["foreign", "Foreign words used in English"],
+    ["names", "Gender neutral first names"],
+    ["choc", "Things that go well with chocolate"],
+    ["crush", "Things people do to impress their crush"]
+]);
+
+/**
+ * Contains all the functionality to randomly generate a word from the given category.
+ * 
+ * @param {prop[]} props - variables being passed into this component
+ * @returns HTML to control the random generation of words. Also displays the word.
+ */
+export default function GetCurrentWord({ category, word }) {
+    // ----- declarations ----- 
+
+    // tell React to update components when answer is changed.
+    const [answer] = useState('');
+
+    // ----- functions ----- 
+
+
+    // ----- html of component -----
+    const currKey = category; 
+    return (
+        <>
+            <h2>Get Random Word Component</h2>
+            <div>Category: {CATEGORIES.get(currKey)}</div>
+            <br />
+            <button onClick={() => {
+                    saveGameState("playing");
+                    
```

Comment:
This section seems incomplete, perhaps you were planning to put a function in here?

Reply:
Eh? No actually haha